e2e/config/e2e.constants.ts

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import { baseSepolia, Chain } from "wagmi/chains";
+
+export const isE2eTestEnabled = process.env.NEXT_PUBLIC_E2E_TEST_ENABLED === "true";
+if (isE2eTestEnabled) console.warn("E2E tests are enabled. This should only be used in a testing environment.");
+export const baseSepoliaE2eForkUrl = "http://127.0.0.1:8545";
```

Comment:
This could be extracted to env vars and extended to e2e test with multiple chains.

Reply:
As a general rule, we prefer to use environment variables for scenarios like this. It makes things much smoother for both local development and when deploying to different environments on Vercel